bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_rdataframe.py

```diff
@@ -74,16 +89,80 @@ def RDataFrameAsNumpy(df, columns=None, exclude=None):
     return py_arrays
 
 
+# Pythonizations for the following methods:
+# - ROOT.RDataFrame.Histo1D
+# - ROOT.RDataFrame.Histo2D
+# - ROOT.RDataFrame.Histo3D
+# - ROOT.RDataFrame.Profile1D
+# - ROOT.RDataFrame.Profile2D
+# Since a lot of their overloads take a ROOT.RDF.T*$DModel (where * can be
+# 'H' and 'Profile', while $ 1, 2 and 3) object as first argument,
+# we implement the possibility to pass the arguments of
+# ROOT.RDF.TH*DModel as a tuple in the first
+# argument of the five methods mentioned before.
+def pythonize_hp(self, partialmethod_list, *args):
```

Comment:
You should be able to define this as `def _histo_profile(self, (original_method_name, model_class), *args):`

Reply:
It complains for invalid syntax

Reply:
This works for me:
```def g(a, (x,y), *b): print(x)```

Reply:
`>>> def g(a, (x,y), *b): print(x)
  File "<stdin>", line 1
    def g(a, (x,y), *b): print(x)
             ^
SyntaxError: invalid syntax`

Reply:
Looks like it works in Python 2 but not in Python 3

Reply:
Yep:
https://stackoverflow.com/questions/20035591/python-cant-define-tuples-in-a-function
Just unpack the tuple at the beginning of the function then, as suggested in that post.